### DISCUSSION TITLE
Why SQANTi3 filter set perc_A_downstream_TTS is [0,59]?

### DISCUSSION THREAD

**Discussion by MR-D-CJ** [ConesaLab/SQANTI3#375] [Q&A] [answered]
Why set "perc_A_downstream_TTS":[0,59], and what is the principle? Greater than 60 Why is this transcript considered to be incorrect? Why is the number of FSM nodes greater than 60

**Comment by carolinamonzo** [comment, marked as answer]
Hi MR-D-CJ, the perc_A_downstream_TTS gives you percent of genomic "A"s in the downstream 20 bp window. If this number is high the 3' end site of this isoform is probably not reliable. It is up to you to decide which is your threshold. We decided on 60% to classify as possible intraprimming since in the original [SQANTI](https://pmc.ncbi.nlm.nih.gov/articles/PMC5848618/) paper they saw A-rich genomic DNA regions downstream from the TTS were a marker for intraprimming.
